doc/docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -1,4 +1,9 @@
 [#neo4j_configuration_system]
+
+[NOTE]
+The Neo4j Streams Plugin running inside the Neo4j database is deprecated and will not be supported after version 4.2 of Neo4j.
```

Comment:
4.3 I think, not 4.2

Reply:
Sorry @moxious I just copy/paste the text in your comment here:
https://github.com/neo4j-contrib/neo4j-streams/pull/482#commitcomment-56258070

I'll replace this and all the all the notes below as well

Reply:
@moxious (I was wondering) given the comment above maybe makes sense to change the notes in this way:

```
[NOTE]
The Neo4j Streams Plugin running inside the Neo4j database is deprecated and will not be supported after version 4.3 (included) of Neo4j.
```